ros2service/ros2service/verb/call.py

```diff
@@ -49,17 +51,23 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '-r', '--rate', metavar='N', type=float,
             help='Repeat the call at a specific rate in Hz')
+        add_qos_arguments(parser, 'services_default')
```

Comment:
I think `add_qos_arguments` should be extended to take `service` and other entities, not statically expecting either `publisher` or `subscription`.

Reply:
No sure I understand this. Could you elaborate your intention, please?

Reply:
I mean,

`def add_qos_arguments(parser: ArgumentParser, subscribe_or_publish: str, default_profile_str):` to `def add_qos_arguments(parser: ArgumentParser, entity_type: str, default_profile_str):`

so that we can use it any entities such as publisher, subscription, services, actions for future use.

Reply:
So far I have 
```
def add_qos_arguments(parser: ArgumentParser, default_profile_str: str):
```
and think this is the best way to go, as it is independent on the entity type.